mipidsi/examples/spi-st7789-rpi-zero-w/src/main.rs

```diff
@@ -0,0 +1,78 @@
+use display_interface_spi::SPIInterface;
+use embedded_graphics::{
+    mono_font::{ascii::FONT_6X10, MonoTextStyle},
+    pixelcolor::Rgb565,
+    prelude::*,
+    text::Text,
+};
+use mipidsi::Builder;
+use rppal::gpio::{Gpio, OutputPin};
+use rppal::hal::Delay;
+use rppal::spi::{Bus, Mode, SlaveSelect, Spi};
+use std::process::ExitCode;
+use std::thread::sleep;
+use std::time::Duration;
+
+const SPI_DC: u8 = 9;
+const SPI_CS: u8 = 1;
```

Comment:
My guess is that there is a mistake in the Pyhton code and it works by accident, because the CS pin is low all the time. To test this theory I would just leave the code as it is and configure GPIO 7 as an high output. If it still works CS is connected to GPIO 1 and if not it is connected to GPIO 7.

> Why would one use pin 1 (3v3)? Is it a mistake in the python code?

Keep in mind that the GPIO pin numbers are different from the physical pin numbers on the 40 pin connector. GPIO 1 is connected to pin 28 (https://pinout.xyz/pinout/pin28_gpio1/#), which is also very close to GPIO 7.

Reply:
You were right once again, thanks! Updated the readme with the pre-requisite